Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/internal/Hole_filling/Triangulate_hole_polyline.h

```diff
@@ -1326,7 +1326,7 @@ triangulate_hole_polyline_with_cdt(const PointRange& points,
   }
 
   FT x = FT(0), y = FT(0), z = FT(0);
-  std::size_t num_normals = 0;
+  FT num_normals = 0;
```

Comment:
`++num_normals` looks strange, for a floating-point type. But that seems valid, if not idiomatic.

Reply:
Can num_normals grow large? If it can, with a lazy FT, it may create a deep tree which we know causes trouble during update_exact and destruction (I had experimented with avoiding the tree when the approximate interval turns out to be a point, but I don't think that ever led to a proper patch and PR). It it cannot, then we might as well use (`unsigned`) `long` or smaller for this counter, or keep size_t but convert it to that shorter type before the division.